offixsdk/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
I will name it offix-offline or offix-client depending on the approach we are taking.